src/services/puzzle/puzzle-formats.ts

```diff
@@ -0,0 +1,21 @@
+import { PuzzleItem } from "../../database/puzzle-db.js";
+
+export function isValidPuzzleItemFormat(obj: PuzzleItem): boolean {
+    if (typeof obj.userId !== "string") {
+        return false;
+    }
+
+    if (typeof obj.teamName !== "string") {
+        return false;
+    }
+
+    if (typeof obj.lastCorrect !== "number") {
+        return false;
+    }
+
+    if (!obj.problemComplete) {
+        return false;
+    }
```

Comment:
This should not be in this function, doesn't change if the format is valid or not.